src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/OperationTestBase.java

```diff
@@ -0,0 +1,47 @@
+package io.stargate.sgv2.jsonapi.service.operation.model.impl;
+
+import com.datastax.oss.driver.api.core.ProtocolVersion;
+import com.datastax.oss.driver.api.core.cql.ColumnDefinition;
+import com.datastax.oss.driver.api.core.cql.ColumnDefinitions;
+import com.datastax.oss.driver.api.core.detach.AttachmentPoint;
+import com.datastax.oss.driver.api.core.type.codec.TypeCodecs;
+import com.datastax.oss.driver.internal.core.cql.DefaultColumnDefinition;
+import com.datastax.oss.driver.internal.core.cql.DefaultColumnDefinitions;
+import com.datastax.oss.protocol.internal.response.result.ColumnSpec;
+import com.datastax.oss.protocol.internal.response.result.RawType;
+import io.stargate.sgv2.jsonapi.api.model.command.CommandContext;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class OperationTestBase {
+  protected final String KEYSPACE_NAME = RandomStringUtils.randomAlphanumeric(16);
+  protected final String COLLECTION_NAME = RandomStringUtils.randomAlphanumeric(16);
+  protected final CommandContext CONTEXT = new CommandContext(KEYSPACE_NAME, COLLECTION_NAME);
+
+  protected ColumnDefinitions buildColumnDefs(List<String> columnNames, List<Integer> typeCodes) {
```

Comment:
Can we use List<Pair<>> instead of 2 array

Reply:
Sure, let me see which Pair we have available.

Reply:
Commons-lang3 has Pair, but also realized that it's even cleaner to just add and use `record` for information.